tests/rest_api_test_case_base.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+import requests
+BASE_URL = "https://www.breakingbadapi.com/api/"
+
+class RestApiTestCaseBase(unittest.TestCase):
```

Comment:
Oh! I see your idea but do not wrap up into the unittest library, use a pure function for that Or you can use your own class to keep methods.